### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by daniela-trustly**
* Add section about handling of  `TrustlyCheckoutEvents` 
 - Focus on redirects and clarify correct method of opening URLs
* Improve code snippet for instantiating TrustlyWKWebView. In the new snippet the webview is embedded in a ViewController for easier navigation between the views
*  Replace URL leading to 404 with correct resource

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,10 @@ Pass your Checkout URL when initialising a new TrustlyWKWebView instance. The Ch
 
 Example usage:
 ```swift
-let trustlyWebView = TrustlyWKWebView(checkoutUrl: trustlyCheckoutURLString, frame: self.view.frame)
-self.view = trustlyWebView 
+let webViewController = UIViewController.init(nibName: nil, bundle: nil)
+let mainView = TrustlyWKWebView(checkoutUrl: checkoutUrl.absoluteString, frame: self.view.bounds)
```
Comment (jennylarsson94) [inline README.md]: Maybe the `checkout.absolutString` can be replace with a placeholder like so `[checkout Url]` or something that's a bit more generic?

File: README.md [hunk 2]
```diff
@@ -48,7 +50,40 @@ class ViewController: UIViewController, TrustlyCheckoutDelegate {
...
+Please note that when rendering the Trustly Checkout from a native app you are required to pass your application’s [URLScheme](https://developer.apple.com/documentation/xcode/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app) as an attribute to the order initiation request. By doing so, Trustly can redirect users back to your app after using external identification apps such as Mobile BankID. You can pass your URLScheme by including it in the "URLScheme" attribute when making an API call to Trustly. [You can read more about it here.](https://developers.trustly.com/emea/docs/ios)
```
Comment (jennylarsson94) [inline README.md:89]: Very nice! Can we add this in the Android README as well?

**Review by jennylarsson94 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (daniela-trustly) [reply README.md:89]: Done https://github.com/trustly/TrustlyAndroidSdk/blob/master/README.md#notes-about-urlscheme

File: README.md (hunk 1 above)
Comment (daniela-trustly) [reply README.md]: Done

**Pull request merged**